azure-pipelines-full-tests.yml

```diff
@@ -13,7 +13,7 @@ pr:
     - '*'
 
 variables:
-- template: .azure-pipelines/templates/variables.yml
+- template: ${{variables['Pipeline.Workspace']}}/.azure-pipelines/templates/variables.yml
```

Comment:
Can we use `$(Pipeline.Workspace)`?

Reply:
If we use `$(Pipeline.Workspace)`, ADO pipeline will report an error:
```
/azure-pipelines.yml: Could not find /$(Pipeline.Workspace)/azure-cli/.azure-pipelines/templates/variables.yml in repository self hosted on https://github.com/ using commit a2544c34c66f64ef3bc97983da82dd4db3595b40. GitHub reported the error, "Not Found"
```

Reply:
To add more info, per https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#understand-variable-syntax, template expression variables (`${{ variables.var }}`) is officially documented.